pom.xml

```diff
@@ -298,13 +298,13 @@
     <dependency>
       <groupId>org.junit-pioneer</groupId>
       <artifactId>junit-pioneer</artifactId>
-      <version>1.7.1</version>
+      <version>2.0.0</version>
```

Comment:
https://mvnrepository.com/artifact/org.junit-pioneer/junit-pioneer
2.0.0 was released on 12/21.
1.7.2 was released on 11/9.
1.7.1 was released on 5/25.

I am confused as I was told the new version was 2.0.0 but maven says it is actually `2.0.0-RC1`.